src/components/BulkEnrollmentPage/CourseSearchResults.jsx

```diff
@@ -39,16 +44,17 @@ export const EnrollButton = ({ row, setSelectedCourseRuns, setModalOpen }) => {
   );
 };
 
+const ExportDataTableContext = () => {
+  const { courses: [, coursesDispatch] } = useContext(BulkEnrollContext);
```

Comment:
My brain is breaking from this definition.
I know the context has `{courses: [courseTableState, coursesDispatch]...}` and based on how we're using `coursesDispatch` later, I am guessing this is saying "get coursesDispatch which is the second item in this list of the courses object in this context"? I just didn't know you could do object retrieval like that (pretty cool if that is the case. Otherwise, I clearly have no idea what's going on here)

Reply:
It is doing exactly that! It's destructuring syntax. It is, for me, **the most hated** part of destructuring syntax, but the linter won't let me have a variable that I don't use, so.
It won't let me link to directly to it, but it's under "Ignoring some return values" 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

Reply:
That's a handy link! The other confusing bit was also the use of the courses key, since normally that'd be a definition (ie `const something = { x: y }`), but here we are just specifying which object to retrieve certain data out of.